uploadscript/reset-database.js

```diff
@@ -0,0 +1,55 @@
+if (process.argv[2].match(/y(es)?/i)) {
```

Comment:
ooh nice writing up this script so we don't have to do manual deletion haha

worth changing the name of the uploadscript package to be more general since it's no longer just for the uploading script? and worth renmaing the script.js file to be upload-script.js?

Reply:
yar